src/data/secondary-options.js

```diff
@@ -470,5 +470,15 @@ export const secondaryOptions = {
       label: 'create a new branch and apply stash',
       usage: 'git stash branch <branch name> <stash id>'
     }
-  ]
+  ],
+
+  cherrypick: [
+    {
+      value: 'origin-branch',
+      label: 'an commit from origin branch into my working branch',
+      usage: 'git cherry-pick <commit-hash> <commit-hash>',
```

Comment:
I was looking at this documentation for [Cherry Pick](https://www.previousnext.com.au/blog/intro-cherry-picking-git) and I think you only need one commit-hash?
```suggestion
      usage: 'git cherry-pick <commit-hash>,
```

Reply:
Multiple commit is also possible if the user want to cherry-pick more than one commit